roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -124,14 +129,18 @@ def local_images(self, images, task_vars):
         ]
 
     def is_image_local(self, image, task_vars):
-        result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
+        """Check if image is already in local docker index."""
+        result = self.execute_module("docker_image_facts",
+                                     module_args={"name": image},
+                                     task_vars=task_vars)
```

Comment:
The named args is fine, but the related problem here is that we ignored `tmp`...

It possibly worked because `task_vars` was not needed by those modules we call, maybe?!